### PR TITLE
chore(deploy): remove feature environment flags

### PR DESCRIPTION
**Pull request by ttbarnes**
## Introduction :pencil2:
- The `feature` environment has been deleted.
- We have some feature related flags in the deployment scripts/YMLs.

## Resolution :heavy_check_mark:
- Remove instances of `feature`.

### CODE REVIEW

File: .github/workflows/deploy.sh [hunk 1]
```diff
@@ -27,7 +27,7 @@ if [ -n "$selection" ]; then
             branch=main
         ############### FEATURE ###############
         elif [ "$destination" = "feature" ]; then
-            branch=main-iterations
+            branch=main
```
Comment (Zainzzkk) [inline .github/workflows/deploy.sh:30]: are we deleting the feature environment? in which case probably best to totally remove this section

Comment (ttbarnes) [reply .github/workflows/deploy.sh:30]: I think it is already deleted, can you double check in Azure please?

Comment (Zainzzkk) [reply .github/workflows/deploy.sh:30]: it is deleted so this can be deleted @abhi-markan

Comment (ttbarnes) [reply .github/workflows/deploy.sh:30]: Thanks @Zainzzkk i've removed `feature` from all deployment related scripts/YML

**Review by abhi-markan — APPROVED** [review]

**Review by Zainzzkk — APPROVED** [review]

**Pull request merged**